2024-05-20 21:12:45,570 INFO    MainThread:21037 [wandb_setup.py:_flush():76] Current SDK version is 0.17.0
2024-05-20 21:12:45,570 INFO    MainThread:21037 [wandb_setup.py:_flush():76] Configure stats pid to 21037
2024-05-20 21:12:45,570 INFO    MainThread:21037 [wandb_setup.py:_flush():76] Loading settings from /Users/maheepchaudhary/.config/wandb/settings
2024-05-20 21:12:45,570 INFO    MainThread:21037 [wandb_setup.py:_flush():76] Loading settings from /Users/maheepchaudhary/pytorch/Projects/concept_eraser_research/DAS_MAT/wandb/settings
2024-05-20 21:12:45,570 INFO    MainThread:21037 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-05-20 21:12:45,570 INFO    MainThread:21037 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-05-20 21:12:45,570 INFO    MainThread:21037 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-05-20 21:12:45,571 INFO    MainThread:21037 [wandb_setup.py:_flush():76] Applying login settings: {}
2024-05-20 21:12:45,571 INFO    MainThread:21037 [wandb_init.py:_log_setup():520] Logging user logs to /Users/maheepchaudhary/pytorch/Projects/concept_eraser_research/DAS_MAT/wandb/run-20240520_211245-idnacqd2/logs/debug.log
2024-05-20 21:12:45,571 INFO    MainThread:21037 [wandb_init.py:_log_setup():521] Logging internal logs to /Users/maheepchaudhary/pytorch/Projects/concept_eraser_research/DAS_MAT/wandb/run-20240520_211245-idnacqd2/logs/debug-internal.log
2024-05-20 21:12:45,571 INFO    MainThread:21037 [wandb_init.py:init():560] calling init triggers
2024-05-20 21:12:45,571 INFO    MainThread:21037 [wandb_init.py:init():567] wandb.init called with sweep_config: {}
config: {}
2024-05-20 21:12:45,571 INFO    MainThread:21037 [wandb_init.py:init():585] re-initializing run, found existing run on stack: uuog5nyi
2024-05-20 21:12:45,572 INFO    MainThread:21037 [wandb_run.py:_finish():2103] finishing run counterfactuals/sae_concept_eraser/uuog5nyi
2024-05-20 21:12:45,577 ERROR   MainThread:21037 [jupyter.py:save_history():437] Run pip install nbformat to save notebook history
2024-05-20 21:12:45,577 INFO    MainThread:21037 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-05-20 21:12:45,577 INFO    MainThread:21037 [wandb_init.py:_jupyter_teardown():448] cleaning up jupyter logic
2024-05-20 21:12:45,577 INFO    MainThread:21037 [wandb_run.py:_atexit_cleanup():2343] got exitcode: 0
2024-05-20 21:12:45,577 INFO    MainThread:21037 [wandb_run.py:_restore():2326] restore
2024-05-20 21:12:45,577 INFO    MainThread:21037 [wandb_run.py:_restore():2332] restore done
2024-05-20 21:12:51,237 INFO    MainThread:21037 [wandb_run.py:_footer_history_summary_info():3994] rendering history
2024-05-20 21:12:51,237 INFO    MainThread:21037 [wandb_run.py:_footer_history_summary_info():4026] rendering summary
2024-05-20 21:12:51,242 INFO    MainThread:21037 [wandb_run.py:_footer_sync_info():3953] logging synced files
2024-05-20 21:12:51,463 INFO    MainThread:21037 [wandb_init.py:init():610] starting backend
2024-05-20 21:12:51,464 INFO    MainThread:21037 [wandb_init.py:init():614] setting up manager
2024-05-20 21:12:51,466 INFO    MainThread:21037 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2024-05-20 21:12:51,466 INFO    MainThread:21037 [wandb_init.py:init():622] backend started and connected
2024-05-20 21:12:51,477 INFO    MainThread:21037 [wandb_run.py:_label_probe_notebook():1328] probe notebook
2024-05-20 21:12:51,477 INFO    MainThread:21037 [wandb_run.py:_label_probe_notebook():1338] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2024-05-20 21:12:51,477 INFO    MainThread:21037 [wandb_init.py:init():711] updated telemetry
2024-05-20 21:12:51,545 INFO    MainThread:21037 [wandb_init.py:init():744] communicating run to backend with 90.0 second timeout
2024-05-20 21:12:53,109 INFO    MainThread:21037 [wandb_run.py:_on_init():2396] communicating current version
2024-05-20 21:12:53,197 INFO    MainThread:21037 [wandb_run.py:_on_init():2405] got version response 
2024-05-20 21:12:53,197 INFO    MainThread:21037 [wandb_init.py:init():795] starting run threads in backend
2024-05-20 21:12:55,775 WARNING MainThread:21037 [wandb_init.py:init():1168] interrupted
Traceback (most recent call last):
  File "/Users/maheepchaudhary/miniforge3/envs/nnsight/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 1164, in init
    return wi.init()
           ^^^^^^^^^
  File "/Users/maheepchaudhary/miniforge3/envs/nnsight/lib/python3.12/site-packages/wandb/sdk/wandb_init.py", line 808, in init
    run_start_result = run_start_handle.wait(timeout=30)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maheepchaudhary/miniforge3/envs/nnsight/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 283, in wait
    found, abandoned = self._slot._get_and_clear(timeout=wait_timeout)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maheepchaudhary/miniforge3/envs/nnsight/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 130, in _get_and_clear
    if self._wait(timeout=timeout):
       ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maheepchaudhary/miniforge3/envs/nnsight/lib/python3.12/site-packages/wandb/sdk/lib/mailbox.py", line 126, in _wait
    return self._event.wait(timeout=timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maheepchaudhary/miniforge3/envs/nnsight/lib/python3.12/threading.py", line 655, in wait
    signaled = self._cond.wait(timeout)
               ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/maheepchaudhary/miniforge3/envs/nnsight/lib/python3.12/threading.py", line 359, in wait
    gotit = waiter.acquire(True, timeout)
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
KeyboardInterrupt
